test/integration_tests/long/test_similarity.py

```diff
@@ -523,3 +543,33 @@ def test_end_to_end_index_scan_should_work_correctly_on_image_dataset_pinecone(
 
             drop_index_query = "DROP INDEX testpineconeindeximagedataset;"
             execute_query_fetch_all(self.evadb, drop_index_query)
+
+    @pytest.mark.skip(reason="Requires running local Milvus instance")
```

Comment:
Is it possible to set up a local Milvus instance for testing in circle ci?

Reply:
Currently trying this out

Reply:
Seems like Milvus Lite (a library to spin up a local instance in Python code) only has an outdated version of Milvus and therefore does not support cosine similarity. Therefore, this doesn't seem possible to do :(